hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -711,6 +711,31 @@ message PrefixInfo {
     optional uint64 updateID = 5;
 }
 
+/**
+ * SnapshotStatus - snapshot states.
+ * Snapshot in one of : active, deleted, or reclaimed state
+ */
+enum SnapshotStatusProto {
+  SNAPSHOT_ACTIVE = 1;
+  SNAPSHOT_DELETED = 2;
+  SNAPSHOT_RECLAIMED = 3;
+}
+
+/**
+ * SnapshotInfo table entry for Bucket/Volume snapshot metadata
+ */
+message SnapshotInfoEntry {
```

Comment:
Suggestion: Can we have just _SnapshotInfo_ instead of _SnapshotInfoEntry_

Reply:
By using `SnapshotInfo` both for the proto and the class, it makes some declarations ambiguous without fully qualifying the objects used.  I too wanted to use `SnapshotInfo` for the proto definition at first.  I avoided the ambiguity  by using a unique identifier (`SnapshotInfoEntry`) for the proto snapshotinfo definition.

As you suggested, I renamed the proto definition to `SnapshotInfo` and fully qualified the use of the proto where needed.  In latest commit.  Thanks.

Reply:
> Can we also add Volume Name and Bucket Name to the Snapshot Info structure?

Added both volume name and bucket name fields to our `SnapshotInfo` table.  Thanks @nandakumar131